test/common/fit_common.py

```diff
@@ -305,13 +307,59 @@ def mkargs(in_args=None):
     arg_parser.add_argument("-v", default=1, type=int,
                             help="Verbosity level of console output, default=1, Built Ins: " +
                                  "0: No debug, " +
-                                 "2: User script output, " +
-                                 "4: rest calls and status info, " +
-                                 "6: other common calls (ipmi, ssh), " +
-                                 "9: all the rest ")
+                                 "2: Display infra.run and test.run DEBUG, " +
+                                 "4: Display test.data (rest calls and status) DEBUG, " +
+                                 "6: infra.data DEBUG (ipmi, ssh), " +
+                                 "9: infra.* and test.* at DEBUG_9 (max output) ")
+    arg_parser.add_argument("-nose-help", default=False, action="store_true", dest="nose_help",
+                            help="display help from underlying nosetests command, including additional log options")
+    # we want to grab the arguments we want, and pass the rest
+    # into the nosetest invocation.
+    parse_results, other_args = arg_parser.parse_known_args(in_args)
+
+    # if 'help' was set, WE to handle it as best we can. We use argparse to
+    # display usage and arguments, and then give nose a shot at printing
+    # things out (if they set that option)
+    if parse_results.help:
+        arg_parser.print_help()
+        if parse_results.nose_help:
+            print
+            print "NOTE: below is the --help output from nosetests."
+            print
+            rcode = _run_nose_help()
+        else:
+            rcode = 0
+        sys.exit(rcode)
+
+    # And if they only did --nose-help
+    if parse_results.nose_help:
+        rcode = _run_nose_help()
+        sys.exit(rcode)
+
+    # now handle mapping -v to infra-logging
+    if parse_results.v >= 9:
+        # Turn them all up to 11.
+        vargs = ['--sm-set-combo-level', 'console*', 'DEBUG_9']
+    elif parse_results.v >= 6:
+        # ends up turning everything up to DEBUG (levels 2, 4, and 6)
+        vargs = ['--sm-set-combo-level', 'console*', 'DEBUG']
+    elif parse_results >= 4:
+        # infra.run and test.* to DEBUG (levels 2 and 4)
+        vargs = ['--sm-set-combo-level', 'console*:(infra.run|test.*)', 'DEBUG']
+    elif parse_results >= 2:
+        # infra and test.run to DEBUG
+        vargs = ['--sm-set-combo-level', 'console*:*.run', 'DEBUG']
+    else:
```

Comment:
Thanks for the updates, tested them out.  For verbosity 1 (default), I request NO debug to go to either test_run.log or test_data.log, I just want INFO.  (I'd prefer this also for infra, but compromise).  As a user,  I want to turn up the debug as I go.  To be starting with it ON at DEBUG(_5) by default,  there is not much room to dial it up.    Debug can start on verbosity 2 and on up to infinity. please. :-)